src/fi_tostr.c

```diff
@@ -619,20 +621,27 @@ static void fi_tostr_cq_event_flags(char *buf, uint64_t flags)
 	fi_remove_comma(buf);
 }
 
+static void fi_tostr_size_t(char *buf, size_t value)
+{
+	strcatf(buf, "%zd", value);
+}
+
 __attribute__((visibility ("default")))
 char *DEFAULT_SYMVER_PRE(fi_tostr)(const void *data, enum fi_type datatype)
 {
 	static char *buf = NULL;
 	uint64_t val64;
 	uint32_t val32;
 	int enumval;
+	size_t valSizeT;
```

Comment:
You've been spending too much time in windows land.  :)  CamelCase isn't used elsewhere in the main portions of the project.